app/k9mail/build.gradle

```diff
@@ -94,8 +84,6 @@ android {
             applicationIdSuffix ".debug"
             testCoverageEnabled rootProject.testCoverage
 
-            minifyEnabled false
```

Comment:
This line was kept intentionally to make it very explicit that minification is not enabled. Please don't remove it.

Reply:
d56eb8d1ba748d1d2c5e6bc153b30e10c989d9ef